content/advocate-with-mau/editable/volunteers-&-vacancies.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Volunteers & Vacancies
+content: From our symposium days to helping children and families through our
+  Lapor Predator chatbot, we could not have accomplished all of it without the
+  support from our dedicated and supportive volunteers. We believe in the power
+  of people and we are looking for youths and individuals of any background that
+  are interested to help out in making a difference. If you're on break, still
+  studying, or simply looking for something to invest your time and energy in,
+  CSA prevention might just be your thing.
+thumbnail: /img/advocate/activity.jpg
+---
+We provide volunteering and job opportunities with different levels of workload and challenges, and we promise there is something for everyone. If this sounds like a great opportunity to you, hop on board to join us, and we shall work together soon.
+
+No postings available currently. But if you are interested to join us, <http://bit.ly/sayamaujoinmau>[](http://localhost:3000/advocate-with-mau/bit.ly/sayamaujoinmau)
```

Comment:
This portion should be static because you need to check if postings are available or not. You can add 'postings' in the frontmatter